extensions/interactions/base_test.py

```diff
@@ -150,7 +150,8 @@ def test_interaction_properties(self):
             'customization_arg_specs', 'is_trainable', 'is_terminal',
             'is_linear', 'rule_descriptions', 'instructions',
             'narrow_instructions', 'needs_summary',
-            'default_outcome_heading', 'can_have_solution'])
+            'default_outcome_heading', 'can_have_solution',
+            'show_nav_submit_button'])
```

Comment:
Still need to modify test_default_interactions_are_valid(), particularly the _INTERACTION_CONFIG_SCHEMA. Always try and find ways to ensure that the standard conventions you're setting up persist into the future.